server/sonar-server/src/main/java/org/sonar/server/component/ws/SearchProjectsAction.java

```diff
@@ -21,17 +21,14 @@
 
 import com.google.common.collect.ImmutableMap;
 import com.google.common.collect.Ordering;
-import java.util.Collections;
-import java.util.LinkedHashMap;
-import java.util.List;
-import java.util.Map;
+
+import java.util.*;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Explicitly import the specific classes needed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2208)